### ISSUE TITLE
Make it clearer in the docs that ResourceLoadEvent == "load"

### ISSUE THREAD

**Issue by vitiral** [koute/stdweb#194]
ResourceLoadEvent links to these javascript docs:

https://developer.mozilla.org/en-US/docs/Web/Events/load

which are just the `load` event.

**Comment by vitiral** [comment]
https://docs.rs/stdweb/0.4.2/src/stdweb/webapi/events/dom.rs.html#53
```
    const EVENT_TYPE: &'static str = "load";
```

**Comment by koute** [comment]
Yeah, I'll probably have to rename those events to something like `UiLoadEvent` (since it derives from `UiEvent`) and create a generic `GenericLoadEvent` which would derive only from `Event`. (In general events like `"load"` or `"abort"` are a mess since there are multiple different events all using the same name but with different type.)
